src/include/osl_pvt.h

```diff
@@ -617,6 +617,14 @@ class Symbol {
     bool everread () const  { return lastread() >= 0; }
     bool everwritten () const { return lastwrite() >= 0; }
     bool everused () const  { return everread() || everwritten(); }
+    // everused_in_group is an even more stringent test -- not only must
+    // the symbol not be used within the shader but it also must not be
+    // used elsewhere in the group, by being connected to something downstream
+    // or used as a renderer output.
+    bool everused_in_group () const {
```

Comment:
Sure. But there's not much to do about that if we haven't yet gotten to the use analysis in the later layer. But, a bit later, if it's discovered that the downstream layer doesn't use that parameter, the connection will be severed, and there will be a second chance post-optimization for layers to be merged.